x/ethbridge/types/claim.go

```diff
@@ -11,6 +11,8 @@ import (
 )
 
 type EthBridgeClaim struct {
+	Chain            string          `json:"chain"`

```

Comment:
I think ChainID should become part of the oracleID used for the oracle and so part of CreateOracleClaimFromEthClaim and the querier that asks for a particular claim.

Reply:
be6ce9d49dac7195bbcf2bf3b5260ebd2afc5715